src/main/java/lotto/service/ParseNumbersService.java

```diff
@@ -0,0 +1,51 @@
+package lotto.service;
+
+import static lotto.constants.ErrorMessage.INPUT_CAN_NOT_BE_BLANK;
+import static lotto.constants.ErrorMessage.LOTTO_CAN_NOT_HAVE_CHARACTER;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import lotto.model.InputWinningNumbers;
+
+public class ParseNumbersService {
+
+    private static final Pattern HAS_CHARACTER_PATTERN = Pattern.compile("[^0-9]");
+    private static final String NUMBERS_DELIMITER = ",";
+    private static final int INCLUDE_AFTER_DELIMITER = -1;
+
+    public ParseNumbersService() {
+    }
+
+    public List<Integer> getWinningLotto(InputWinningNumbers inputWinningNumbers) {
+        List<String> winningNumbers = Arrays.stream(inputWinningNumbers.get()
+                        .split(NUMBERS_DELIMITER, INCLUDE_AFTER_DELIMITER))
+                .map(String::strip).toList();
+
+        validate(winningNumbers);
+
+        return winningNumbers.stream().mapToInt(Integer::parseInt)
+                .boxed()
+                .collect(Collectors.toList());
```

Comment:
현재 언박싱 후 다시 박싱 하는걸로 보여지는데
```suggestion
        return winningNumbers.stream()
                .map(Integer::valueOf)
                .collect(Collectors.toList());
```
이 방법으로 String을 Integer로 직접 변환해보는 것은 어떨까요!

Reply:
오호 좋은 조언 감사합니다! Stream에 익숙해지려면 한참 멀은 것 같아요 😞